src/Analyser/ResultCache/ResultCacheManager.php

```diff
@@ -605,6 +603,27 @@ private function save(
 		);
 	}
 
+	/**
+	 * @param mixed[]|null $projectConfig
+	 */
+	private function getCacheFilePath(?string $resultCacheName, ?array $projectConfig): string
+	{
+		if ($resultCacheName !== null) {
+			return $this->tempResultCachePath . '/' . $resultCacheName . '.php';
+		}
+
+		if ($projectConfig !== null &&
+			array_key_exists('parameters', $projectConfig) &&
+			!array_key_exists('resultCachePath', $projectConfig['parameters']) &&
+			(!array_key_exists('tmpDir', $projectConfig['parameters']) || $projectConfig['parameters']['tmpDir'] === null)
+		) {
+
+			return dirname($this->cacheFilePath) . '/resultCache-' . md5(Neon::encode([$this->analysedPaths, $this->bootstrapFiles])) . '.php';
```

Comment:
only thing I am not 100% sure: what needs to be here in this hash.

maybe just the `$this->analysedPaths` are already enough, since these are the absolute paths within the system..?